_build_xtt.py

```diff
@@ -20,8 +20,8 @@
 
 
 # Note: Reset POST to 0 on version bump
-XTT_VERSION = 'v0.9.2'
-POST        = '2'
+XTT_VERSION = 'v0.10.0'
+POST        = '1'
```

Comment:
Reset `POST` to 0, not 1.

Reply:
Does this not track the deb version?

Reply:
No, it tracks the XTT version. 

The POST here is for the Python packaging revision.  The "REV" in a Debian package is for the Debian packaging revision. Those move independently  (depending on how many mistakes I make while packaging for a particular platform!).

(Debian revisions appear to be 1-indexed, while Python packaging revisions are 0-indexed )

Reply:
Gotcha

Done